sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/AnalyzeBatchActionsOperationDetail.java

```diff
@@ -0,0 +1,196 @@
+                    int actionsInProgress) {
+                    operationDetail.setActionsInProgress(actionsInProgress);
+                }
+
+                @Override
+                public void setActionsSucceeded(AnalyzeBatchActionsOperationDetail operationDetail,
+                    int actionsSucceeded) {
+                    operationDetail.setActionsSucceeded(actionsSucceeded);
+                }
+
+                @Override
+                public void setActionsInTotal(AnalyzeBatchActionsOperationDetail operationDetail, int actionsInTotal) {
+                    operationDetail.setActionsInTotal(actionsInTotal);
+                }
+
+                @Override
+                public void setExpiresAt(AnalyzeBatchActionsOperationDetail operationDetail,
+                    OffsetDateTime expiresAt) {
+                    operationDetail.setExpiresAt(expiresAt);
+                }
+
+                @Override
+                public void setCreatedAt(AnalyzeBatchActionsOperationDetail operationDetail,
+                    OffsetDateTime createdAt) {
+                    operationDetail.setCreatedAt(createdAt);
+                }
+
+                @Override
+                public void setLastModifiedAt(AnalyzeBatchActionsOperationDetail operationDetail,
+                    OffsetDateTime lastModifiedAt) {
+                    operationDetail.setLastModifiedAt(lastModifiedAt);
+                }
+            });
+    }
+
+    /**
+     * Gets the operationId property of the {@link AnalyzeBatchActionsOperationDetail}.
+     *
+     * @return the operationId property of the {@link AnalyzeBatchActionsOperationDetail}.
+     */
+    public String getOperationId() {
+        return operationId;
+    }
+
+    /**
+     * Gets the displayName property of the {@link AnalyzeBatchActionsOperationDetail}.
+     *
+     * @return the displayName property of the {@link AnalyzeBatchActionsOperationDetail}.
+     */
+    public String getDisplayName() {
+        return displayName;
+    }
+
+    /**
+     * Gets the failed number of actions.
+     *
+     * @return the failed number of actions.
+     */
+    public int getActionsFailed() {
+        return actionsFailed;
+    }
+
+    /**
+     * Gets the in-progress number of actions.
+     *
+     * @return the in-progress number of actions.
+     */
+    public int getActionsInProgress() {
+        return actionsInProgress;
+    }
+
+    /**
+     * Gets the successfully completed number of actions.
+     *
+     * @return the successfully completed number of actions.
+     */
+    public int getActionsSucceeded() {
+        return actionsSucceeded;
+    }
+
+    /**
+     * Gets the total number of actions.
+     *
+     * @return the total number of actions.
+     */
+    public int getActionsInTotal() {
+        return actionsInTotal;
+    }
+
+    /**
+     * Gets the created time of a action.
```

Comment:
```suggestion
     * Gets the created time of an action.
```

Same in other places